test/User-test.js

```diff
@@ -0,0 +1,149 @@
+import { expect } from "chai";
+import Recipe from "../src/classes/Recipe";
+import User from "../src/classes/User";
+
+describe("User", () => {
+  it("Should be a function", () => {
+    expect(User).to.be.a("function");
+  });
+
+  let user1;
+  let user2;
+  let user3;
+  let recipe1;
+  let recipe2;
+  let recipe3;
+
+  beforeEach(() => {
+    user1 = new User({
+      name: "Josh",
+      id: 28,
+      pantry: [
+        {
+          ingredient: 11297,
```

Comment:
hmmm....I wish this key was called `ingredientId` instead of `ingredient`, since it holds a number that matches the `id` of an ingredient from `ingredientsData`

Reply:
Thanks, changes were made.